tasks/main.yml

```diff
@@ -1,6 +1,11 @@
 - name: Including distribution-specific packages
   include_vars: "{{ ansible_os_family | lower }}.yml"
 
+- name: Including Proxmox-specific packages
+  include_vars: proxmox.yml
+  when:
+    - sp_proxmox_node | bool
```

Comment:
Having a boolean host variable to determine if a host is part of a group of hosts is sub-optimal IMO. I'd suggest checking if a host is part the group `proxmox_hv`, e.g.

Reply:
Well, depends on how you define a group... I can change it to look for group membership, all the same :)

Reply:
I prefer having a separate group to designate hosts which are Proxmox hypervisors.